test/support/cdp_utils.rb

```diff
@@ -63,7 +63,7 @@ def connect_to_cdp_server
       MSG
     end
 
-    TIMEOUT_SEC = (ENV['RUBY_DEBUG_TIMEOUT_SEC'] || 10).to_i
+    TIMEOUT_SEC = (ENV['RUBY_DEBUG_TIMEOUT_SEC'] || 30).to_i
```

Comment:
why do you extend the timeout?
this extension will introduce huge slowdown on some cases.

Reply:
I found the test runner was timing out on GitHub actions even in HEAD.